src/globals/data/colors.js

```diff
@@ -1,103 +1,108 @@
-    shortDescription: 'sequential default 02',
-  },
-  'sequential-default-03': {
-    name: 'sequential-default-03',
-    hex: '#005F9B',
-    scss: '$color__sequential-default-03',
-    type: 'sequential-default',
-    shortDescription: 'sequential default 03',
-  },
-  'sequential-default-04': {
-    name: 'sequential-default-04',
-    hex: '#0780C0',
-    scss: '$color__sequential-default-04',
-    type: 'sequential-default',
-    shortDescription: 'sequential default 04',
-  },
-  'sequential-default-05': {
-    name: 'sequential-default-05',
-    hex: '#4AA1E2',
-    scss: '$color__sequential-default-05',
-    type: 'sequential-default',
-    shortDescription: 'sequential default 05',
-  },
-  'sequential-default-06': {
-    name: 'sequential-default-06',
-    hex: '#73C3FE',
-    scss: '$color__sequential-default-06',
-    type: 'sequential-default',
-    shortDescription: 'sequential default 06',
-  },
-  'sequential-default-07': {
-    name: 'sequential-default-07',
-    hex: '#9DEAFF',
-    scss: '$color__sequential-default-07',
-    type: 'sequential-default',
-    shortDescription: 'sequential default 07',
-  },
-  'sequential-secondary-01': {
-    name: 'sequential-secondary-01',
-    hex: '#367a96',
-    scss: '$color__sequential-secondary-01',
-    type: 'sequential-secondary',
-    shortDescription: 'sequential secondary 01', 
-  },
-  'sequential-secondary-02': {
-    name: 'sequential-secondary-02',
-    hex: '#319298',
-    scss: '$color__sequential-secondary-02',
-    type: 'sequential-secondary',
-    shortDescription: 'sequential secondary 02', 
-  },
-  'sequential-secondary-03': {
-    name: 'sequential-secondary-03',
-    hex: '#33aa96',
-    scss: '$color__sequential-secondary-03',
-    type: 'sequential-secondary',
-    shortDescription: 'sequential secondary 03', 
-  },
-  'sequential-secondary-04': {
-    name: 'sequential-secondary-04',
-    hex: '#41c196',
-    scss: '$color__sequential-secondary-04',
-    type: 'sequential-secondary',
-    shortDescription: 'sequential secondary 04', 
-  },
-  'sequential-secondary-05': {
-    name: 'sequential-secondary-05',
-    hex: '#5fd79a',
-    scss: '$color__sequential-secondary-05',
-    type: 'sequential-secondary',
-    shortDescription: 'sequential secondary 05', 
-  },
-  'sequential-secondary-06': {
-    name: 'sequential-secondary-06',
-    hex: '#89eaa8',
-    scss: '$color__sequential-secondary-06',
-    type: 'sequential-secondary',
-    shortDescription: 'sequential secondary 06', 
-  },
-  'sequential-secondary-07': {
-    name: 'sequential-secondary-07',
-    hex: '#bdf9c4',
-    scss: '$color__sequential-secondary-07',
-    type: 'sequential-secondary',
-    shortDescription: 'sequential secondary 07', 
-  },
+  main: {
+    name: 'main',
+    hex: '#0A6EB4',
+    scss: '$color__main',
+    type: 'communications',
+    },
+    'sequential-default-07': {
+      name: 'sequential-default-07',
+      hex: '#9DEAFF',
+      scss: '$color__sequential-default-07',
```

Comment:
We can replace the scss colors by css colors in the future

Reply:
sure